gui/elements/_window.py

```diff
@@ -7,7 +7,12 @@ class Window:
     """Application window.
     """
     def __init__(self):
-        self._qt = QtWindow()
+        self._qt_window = QMainWindow().__init__()
```

Comment:
as stated in person, I would prefer this as just `_qt` since the "window" part is implied

@Napari/gui-design what are your thoughts?

Reply:
The simpler name seems cleaner to me.

Reply:
As a Qt noob, I kinda like `_qt_window`. It depends how often this will be typed, though.

Reply:
I prefer more expressive variable names. It is helpful when you read what you wrote after some time.

Reply:
We are splitting hairs people...
Longer (with reason) and more expressive names are almost always better.